src/elements/element.rectangle.js

```diff
@@ -206,7 +206,15 @@ module.exports = Element.extend({
 
 	getArea: function() {
 		var vm = this._view;
-		return vm.width * Math.abs(vm.y - vm.base);
+		var area;
```

Comment:
we don't really need this variable. we could just return the results directly

Reply:
Updated!